core/src/main/java/org/geysermc/geyser/inventory/click/ClickPlan.java

```diff
@@ -58,11 +59,17 @@ public final class ClickPlan {
     private final InventoryTranslator translator;
     private final Inventory inventory;
     private final int gridSize;
+    private final boolean handleBookRecipe;
 
     public ClickPlan(GeyserSession session, InventoryTranslator translator, Inventory inventory) {
+        this(session, translator, inventory, false);
+    }
+
+    public ClickPlan(GeyserSession session, InventoryTranslator translator, Inventory inventory, boolean handleBookRecipe) {
         this.session = session;
         this.translator = translator;
         this.inventory = inventory;
+        this.handleBookRecipe = handleBookRecipe;
```

Comment:
Make this specific to the clone book recipe, e.g. possibly `cloneBookRecipe`. Also add javadoc explaining what it is for